### PR TITLE
test healthcheckReducer

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/__tests__/store/reducers/healthCheck.reducer.test.js [hunk 1]
```diff
@@ -7,77 +7,164 @@ import {
...
 
-        expect(result.isFetching).toBe(true);
+            expect(result.isFetching).toBe(true);
+            expect(touchedReducerProps(result)).toBe(1);
+        });
     });
 
-    it('returns isFetching', () => {
-        const result = healthCheckRducer(initialState, { type: FETCH_HEALTHCHECK_START });
+    describe('FETCH_HEALTHCHECK_START', () => {
```
Comment (rafaeljesus) [inline src/__tests__/store/reducers/healthCheck.reducer.test.js]: sorry for the dumb question, is this a new style of describing tests?

Comment (ivanzusko) [reply src/__tests__/store/reducers/healthCheck.reducer.test.js]: that is just for the more prettier outcome:

image

Comment (rafaeljesus) [reply src/__tests__/store/reducers/healthCheck.reducer.test.js]: 😮 in fact it is better to read on the console

Comment (ivanzusko) [reply src/__tests__/store/reducers/healthCheck.reducer.test.js]: But this is snapshot of the `console`

**Pull request merged**